README.md

```diff
@@ -109,15 +109,16 @@ Recommended practices
     in the `udf_js` directory
     - The `udf_legacy/` directory is an exception which must only contain
       compatibility functions for queries migrated from Athena/Presto.
-  - Functions must be named with a prefix of `<dir_name>_` so all functions
-    in `udf/*.sql` must start with `udf_`
-    - The final function in a file must be named as
-      `<dir_name>_<file_name_without_suffix>` so `udf/mode_last.sql` must
-      define a function `udf_mode_last`
-- Functions must be defined as temporary using `CREATE TEMP FUNCTION` syntax
-  - We provide tooling in `scripts/publish_persistent_udfs` for converting
-    these definitions to persistent UDFs (temporary UDF `udf_mode_last` is
-    published as persistent UDF `udf.mode_last`)
+- Functions must be defined as [persistent UDFs](https://cloud.google.com/bigquery/docs/reference/standard-sql/user-defined-functions#temporary-udf-syntax)
+  using `CREATE OR REPLACE FUNCTION` syntax
+  - Function names must be prefixed with a dataset of `<dir_name>.` so, for example, 
+    all functions in `udf/*.sql` are part of the `udf` dataset
+    - The final syntax for creating a function in a file will look like
+      `CREATE OR REPLACE FUNCTION <dir_name>.<file_name>`
+  - We provide tooling in `scripts/publish_persistent_udfs` for
+    publishing these UDFs to BigQuery
+    - Changes made to UDFs need to be published manually in order for the
+      dry run CI task to pass
```

Comment:
I'm guessing that that is the case here?

Reply:
yes